### ISSUE TITLE
Manifest Not Found

### ISSUE THREAD

**Issue by cjkoepke** [philipnewcomer/wpdocker#131]
I get this error when trying to boot `pilothouse`, even after uninstalling/reinstalling, updating, etc.

`ERROR: for php7-xdebug  b'manifest for pilothouseapp/php:7-xdebug-dev not found: manifest unknown: manifest unknown'`

**Comment by cjkoepke** [comment]
More context:

```bash
Pulling mailcatcher  ... done
Pulling mysql        ... done
Pulling nginx        ... done
Pulling phpmyadmin   ... done
Pulling redis        ... done
Pulling php70        ... done
Pulling php70-xdebug ... done
Pulling php71        ... done
Pulling php71-xdebug ... done
Pulling php73        ... done
Pulling php73-xdebug ... done
Pulling php72        ... done
Pulling php72-xdebug ... done
Pulling php7         ... error
Pulling php7-xdebug  ... error

ERROR: for php7-xdebug  b'manifest for pilothouseapp/php:7-xdebug-dev not found: manifest unknown: manifest unknown'

ERROR: for php7  b'manifest for pilothouseapp/php:7-dev not found: manifest unknown: manifest unknown'
ERROR: manifest for pilothouseapp/php:7-xdebug-dev not found: manifest unknown: manifest unknown
manifest for pilothouseapp/php:7-dev not found: manifest unknown: manifest unknown
+ pilothouse@0.17.0
added 214 packages from 120 contributors in 12.562s
```

**Comment by cjkoepke** [comment]
Relevant piece of information: this only appears to be a problem when I specify using 7.0 on a site. From my `config.yml` file:

```yml
default_php_version: 7.0
```

**Comment by philipnewcomer** [comment]
@cjkoepke this should be resolved in `0.17.1`.
